tests/test_mesh.py

```diff
@@ -62,6 +62,11 @@ def test_trim(self):
         self.mesh.trim(torch.zeros(1, 1, 2))
         self.mesh.trim(torch.zeros(1, 1, 2), True)
 
+    def test_save_and_load(self):
+        self.mesh.save(os.path.join(self.save_dir, 'test_mesh.json'))
+        saved_mesh = self.mesh.load(os.path.join(self.save_dir, 'test_mesh.json'))
```

Comment:
So is the load path the same as save path? You seem to be appending some stuff to the save path when saving, so I'm not sure how this can work.